calibrators/docs/tutorials/marker_radar_lidar_calibrator.md

```diff
@@ -0,0 +1,237 @@
+
+## Data preparation
+
+Please download the data (rosbag) from [here](https://drive.google.com/drive/folders/1G1n2AV47gbQFKQtlzbHJPeV10m5_uMMk).
+
+The rosabg includes three different topics: `object_raw`, `pointcloud_raw`, and `tf_static`.
+
+## Environment preparation
+
+### Overall calibration environment
+
+The required space for calibration depends on the vehicle and sensors used. Generally, an open space without any other objects is an ideal calibration environment. During the calibration, please always make sure that reflectors are detected by both radar and lidar. The user could always check whether the reflectors appear in the `rviz` with the corresponding topics.
+
+### Radar reflector
+
+It is recommended to utilize a tripod to adjust the height of the radar reflector and also modify its center to align with the radar sensor. More information about the radar reflector can be found in the [general documentation](../../marker_radar_lidar_calibrator/README.md#radar-reflector).
+
+## Launching the tool
+
+In this tutorial, we take the X2 project as an example.
+First, run the sensor calibration manager:
+
+```text
+ros2 run sensor_calibration_manager sensor_calibration_manager
+```
+
+In `project`, select `x2`, and in `calibrator`, select `marker_radar_lidar_calibrator`. Then, press `Continue`.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/menu1.jpg" alt="menu1">
+</p>
+
+A menu titled `Launcher configuration` should appear in the UI, and the user may change any parameter he deems convenient.
+For this tutorial, we modify the default value `radar_name` from `front_left` to `front_center`. After configuring the parameters, click `Launch`.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/menu2.jpg" alt="menu2">
+</p>
+
+The following UI should be displayed. When the `Calibrate` button becomes available, click it.
+If it does not become available, it means that either the required `tf` or services are not available. In this case, since the `tf` are published by the provided rosbag, run the bag with the command `ros2 bag play radar_lidar.db3 --clock -r 0.5` and click the `Calibrate` button.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/menu3.jpg" alt="menu3">
+</p>
+
+### Extract background model
+
+Once the user starts running the tutorial rosbag, the pointcloud should appear in `rviz` as shown in the example below. Press the `Extract Background Model button` in the UI to start extracting the background.
+
+Note that the user should put the radar reflector out of the calibration area before clicking the button. Ideally, there shouldn't be any moving objects like humans and vehicles in the calibration area.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/rviz1.jpg" alt="rviz1" width="500">
+</p>
+
+Once the user clicks the button, it will show like the image below.
```

Comment:
It will show does not sound correct / natural.
Maybe something along the lines of
"the result will appear as shown in the image below"
is better. It probably applies to more places

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/4786ae7e7a15477e6592b62a4a0be15546565d38